formats/table/index.js

```diff
@@ -316,6 +324,12 @@ class TableRow extends BaseRow {
 
     this.childFormatName = 'table';
   }
+
+  deleteAt(index, length) {
```

Comment:
deleteAt is copypast in two places. Can common part be moved to a separate function?